### PR TITLE
Production line - be able to add a product to a production line when in confirmation page, mark stuff completed, production line efficiency report

### PR DESCRIPTION
**Pull request by Lucytheanimefan**
@amanocha Modifications to your code! Please check it out!

### CODE REVIEW

File: routes/production_lines.js [hunk 1]
```diff
@@ -176,4 +195,19 @@ router.post('/delete/:id', function(req, res, next) {
...
+/**
+ * TODO
+ * Req 7.3.3
+ * Managers should be able to select a production and mark it completed, viewing the resulting lot number (req 7.2.7) and causing the product to enter the appropriate inventory (req 7.2.8 for intermediates and 7.2.9 for final products).
+ * @param  {[type]} req   [description]
+ * @param  {[type]} res   [description]
+ * @param  {[type]} next) {             } [description]
+ * @return {[type]}       [description]
+ */
+router.post('/mark_completed/:id', function(req, res, next) {
```
Comment (Lucytheanimefan) [inline routes/production_lines.js]: @dy46 Marking production lines as complete.

File: helpers/formula.js [hunk 2]
```diff
@@ -256,7 +256,9 @@ module.exports.createListOfTuples = function(formulaName, amount) {
         newTuple['amount'] = units*parseFloat(tuple['quantity']);
         total.push(newTuple);
       }
-      resolve(total);
+      console.log('Helper formula!');
+      console.log(formula);
+      resolve({'total':total, 'formula':formula});
```
Comment (Lucytheanimefan) [inline helpers/formula.js]: @amanocha  Returning a different format because my functions use formula id, not name.

File: helpers/ingredients.js [hunk 3]
```diff
@@ -137,7 +137,7 @@ module.exports.decrementAmountForProduction = function(id, amount, formula, form
       }
       for (let lot of lotsConsumed) {
         var time = Date.now() - lot.timestamp;
-        await Freshness.updateReport(lot.ingID, lot.name, lot.amount, time);
+        await Freshness.updateFreshnessReport(lot.ingID, lot.name, lot.amount, time);
```
Comment (Lucytheanimefan) [inline helpers/ingredients.js]: @dy46 Any other name changes I'm unaware of?

File: routes/production_lines.js (hunk 1 above)
Comment (dy46) [reply routes/production_lines.js]: if you want you can do a unique id generator for lot numbers, right now i'm just getting a random number between two large numbers, which isn't super good

File: helpers/ingredients.js (hunk 3 above)
Comment (dy46) [reply helpers/ingredients.js]: if you pulled from master it should be alright

**Pull request merged**